tests/functional/test_wheel.py

```diff
@@ -193,7 +193,7 @@ def test_pip_wheel_fail_cause_of_previous_build_dir(
     result = script.pip(
         'wheel', '--no-index', '--find-links=%s' % data.find_links,
         '--build', script.venv_path / 'build',
-        'simple==3.0', expect_error=True,
+        'simple==3.0', expect_error=True, expect_temp=True,
```

Comment:
The cache should be cleaning up after itself even if this happens?

Reply:
Ditto here for what I said below.

Reply:
Ditto here for what I said below. ;)